docs/en/operations/settings/settings.md

```diff
@@ -761,7 +761,7 @@ Default value: 0.
 
 ## max_network_bandwidth {#settings-max_network_bandwidth}
 
-Limits speed of data exchange over the network in bytes per second. This setting applies for every individual query.
+Limits the speed that data is exchanged at over the network in bytes per second. This setting applies to every query.
```

Comment:
This version sounds misleading. The only real issue previous one had is "applies for -> is applied to".